api/src/pcapi/routes/backoffice/templates/offerer/get.html

```diff
@@ -154,7 +154,10 @@ <h2 class="card-title text-primary">{{ links.build_offerer_name_to_pc_pro_link(o
             <div class="col-4">
               {% if has_permission("CONNECT_AS_PRO") %}
                 <p class="mb-1">
-                  {{ links.build_connect_as_pseudo_link(offerer.id, "offerer", "/offres?structure="~offerer.id~"&from-bo=true", "Offres associées ", "fw-bold link-primary") }}
+                  {{ links.build_connect_as_pseudo_link(offerer.id, "offerer", "/offres?structure="~offerer.id~"&from-bo=true", "Offres individuelles ", "fw-bold link-primary") }}
+                </p>
+                <p class="mb-1">
```

Comment:
Je trouve les liens trop espacés : 
![image](https://github.com/user-attachments/assets/eaf21960-0243-47dc-ac2a-98a03a7c1374)
Je propose : 
```suggestion
                <br/>
```
pour avoir : 
![image](https://github.com/user-attachments/assets/ac1161e4-39ae-46e3-affa-ffa55c69bd02)